### DISCUSSION TITLE
Tmux status changed after I updated to the lastest version

### DISCUSSION THREAD

**Discussion by mau-mauricelim** [catppuccin/tmux#425] [Help] [answered]
I was on 9ee1695d757c16e2f236858b8d3f88be9fb666fa with the following statuses:
![image](https://github.com/user-attachments/assets/8fdfe660-261f-439e-bdbe-31e16c85337b)
On the left, there is the **window base path** - this is what I want.

After I updated to c8a2d1ae9649aa904960bef1516ea2ff9a3e6ad0, the behaviour is lost.
Using the defaults:
![image](https://github.com/user-attachments/assets/a041f17a-53cd-47df-996f-30b8c2b0ee0b)
On the left, it is showing **$HOST** instead - this is not what I want.

I tried using the [Recommended Default Configuration](https://github.com/catppuccin/tmux?tab=readme-ov-file#recommended-default-configuration):
![image](https://github.com/user-attachments/assets/f9898dfb-832c-47e9-ab4e-242fbb2b0de9)
Again, on the left, it is showing **$HOST** instead - this is not what I want.

What can I change in the tmux config to get back the desired behavior?

**Comment by kjnsn** [comment, marked as answer]
So the default now uses the pane title, which means that applications can set the window status titles to whatever they want. It looks like you are using zsh, see https://github.com/ohmyzsh/ohmyzsh/wiki/Settings#automatic-title for how you can customise the title there. fish sets the title automatically

Otherwise just do this:

```
set -g @catppuccin_window_current_text " #{b:pane_current_path}"
set -g @catppuccin_window_text " #{b:pane_current_path}"
```

**Reply by mau-mauricelim** [reply]
That's great thanks! Secondly, do you know how can I remove the separator spacing on the status left?

If you see my first picture, there is no separator spacing, while the second picture, there is a separator spacing between the windows (bottom-left).

**Reply by kjnsn** [reply]
This is a tmux option itself `window-status-separator`, take a look at https://man7.org/linux/man-pages/man1/tmux.1.html#OPTIONS
